pkgs/applications/networking/mumble/default.nix

```diff
@@ -114,6 +114,7 @@ let
         "-D bundled-opus=OFF"
         "-D bundled-speex=OFF"
         "-D bundle-qt-translations=OFF"
+        # "-D bundled-renamenoise=OFF" # not packaged in nixpkgs yet
```

Comment:
It is likely that they switch back to upstream. Please remove this commit.

Reply:
As far as I understand the mumble source, they have [the rename noise library vendored as a git submodule](https://github.com/mumble-voip/mumble/tree/master/3rdparty), and use that by default.

So I'm not sure I understand your comment about mumble switching back to upstream?

I wanted to document that this flag is available, but not usable yet because the library is not yet packaged in nixpkgs. Why don't you like it?

Reply:
He means they switched back to upstream rnnoise in mumble master which is in nixpkgs. See https://github.com/mumble-voip/mumble/pull/6607
This also means renamenoise will probably never get packaged as the only user of  (as far as I know) no longer depends on it.

Reply:
Good point, I have noted that as a comment to make the next update easier.